pkgs/tools/misc/phoronix-test-suite/default.nix

```diff
@@ -19,6 +19,10 @@ stdenv.mkDerivation rec {
     --prefix PATH : ${stdenv.lib.makeBinPath [ gnumake gcc ]}
   '';
 
+  passthru.tests = {
+    tests = callPackage ./tests.nix { };
```

Comment:
`simple-execution` or `run-dummy-command` might be better.

Reply:
yes, that's more explicit. but also longer...

can the file still be "tests.nix" when it's the only tests file? then you see directly which file are the tests in the directory